test/jdk/sun/security/tools/keytool/i18n.java

```diff
@@ -23,17 +23,410 @@
 
 /*
  * @test
- * @bug 4348369 8076069
- * @summary keytool not i18n compliant
+ * @bug 4348369 8076069 8294994
+ * @summary keytool i18n compliant
  * @author charlie lai
- * @run main/manual i18n
+ * @modules java.base/sun.security.tools.keytool
+ * @library /test/lib
+ * @run main/manual/othervm i18n en
  */
 
-import java.nio.file.Path;
+/*
+ * @test
+ * @bug 4348369 8076069 8294994
+ * @summary keytool i18n compliant
+ * @author charlie lai
+ * @modules java.base/sun.security.tools.keytool
+ * @library /test/lib
+ * @run main/manual/othervm i18n de
+ */
+
+/*
+ * @test
+ * @bug 4348369 8076069 8294994
+ * @summary keytool i18n compliant
+ * @author charlie lai
+ * @modules java.base/sun.security.tools.keytool
+ * @library /test/lib
+ * @run main/manual/othervm i18n ja
+ */
+
+/*
+ * @test
+ * @bug 4348369 8076069 8294994
+ * @summary keytool i18n compliant
+ * @author charlie lai
+ * @modules java.base/sun.security.tools.keytool
+ * @library /test/lib
+ * @run main/manual/othervm i18n zh CN
+ */
```

Comment:
Do you need to triplicate these `@test` tags? Would 3-lines of `@run` suffice?
Also setting the locale by `-Duser.language/country` and `getProperty` them in the main would be preferable to passing them as the test case arguments.

Reply:
Sure, I can make a change to use the system property. 
Regarding your first question, a test with multiple 'run' fails and terminates on the first test failure. So I would prefer to have multiple tests rather than multiple 'run' in a single test that it allows jtreg to run all the tests independently.